DropDownBox: update demo descriptions (#27546)
DevExpress/DevExtreme: apps/demos/Demos/DropDownBox/MultipleSelection/description.md

> DropDownBox is one of multiple DevExtreme drop-down editors. To learn how to choose a DevExtreme drop-down editor, refer to the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).

Reviewer: ```suggestion
DropDownBox is one of multiple DevExtreme drop-down editors. For information on which DevExtreme drop-down editor you should use, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/
```

> DropDownBox is an advanced drop-down editor that allows you to embed any content, including other components.

Reviewer 2: This is probably OK. However, what do you think about the following version? An ability to include other component already implies that we have very few limitations regarding possible content...

```suggestion
DropDownBox is an advanced editor whose drop-down window can include other components.
```

> DevExtreme ships with multiple drop-down editors. To find out what editor is suitable for your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).

Reviewer 2: ```suggestion
DevExtreme ships with multiple other drop-down editors. To find out what editor is suitable for your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```

> DevExtreme ships with multiple other drop-down editors. To find out what editor is suitable for your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).

Reviewer: ```suggestion
DevExtreme ships with multiple drop-down editors. To find out which editor best suits your task, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).
```

> DevExtreme offers multiple drop-down editor options. To find out which editor you should use, review the following article: [How to Choose a Drop-Down Editor](/Documentation/Guide/UI_Components/Lookup/Choose_a_Drop-Down_Editor/).

Reviewer 2: Again, this is probably OK. However, this entry does not look consistent with the previous content. For example...

> DevExtreme offers multiple drop-down editor options.

Of what drop-down editor options do we speak here? Do we speak of options applicable to the `DropDownBox` we actually describe here? Do we speak of options common to all drop-down editors? Do we speak of other drop-down editors shipped with DevExtreme?

I haven't read the article you are referring to (How to Choose a Drop-Down Editor), however, its name implies that there we speak of other/all available drop-down editor types. If that's the case, the first sentence

> DevExtreme offers multiple drop-down editor options.

does not solve the task at hand.

Still, even if we find a proper solution for this conundrum, the non-consistency problem remains. At first, we speak of `DropDownBox`, and then all of the sudden start speaking of other editors available and then switch back to `DropDownBox` out of the blue.

If you that need to refer to the topic on other drop-down editors, you should've probably moved it after the next paragraph...

> If the UI component embedded into the DropDownBox allows multiple selection, synchronize the DropDownBox value with the selected items. [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) are the same for every selection mode.

Reviewer 2: First question I'm having at this point -- can I embed only one UI component into the `DropDownBox` component's drop-down window?

> [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) are the same for every selection mode.

Of what selection modes do we speak? It's not obvious from my standpoint.

This paragraph could have looked as follows:

```suggestion
If the UI component embedded into the DropDownBox component's drop-down window allows multiple selection, synchronize the component's value with the state of selected items as demonstrated in the following topic: [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/).
```

However, you can probably come up with something better than that.

> If the UI component embedded into the DropDownBox component's drop-down window allows multiple selection, synchronize the component's value with the state of selected items as demonstrated in the following demo: [Synchronization instructions](/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/).

Reviewer 2: > synchronize the component's value with the state of selected items as demonstrated in the following **demo**:

"Synchronization Instructions" is a strange name for a demo... It rather befits a documentation topic than a demo...

<!--split-->

> In this demo, the DataGrid's **selection**.[mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#mode) and TreeView's [selectionMode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#selectionMode) properties are used to enable multiple selection.

Reviewer 2: Avoidable passive voice.

```suggestion
This demo uses the DataGrid's **selection**.[mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#mode) and TreeView's [selectionMode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#selectionMode) properties to enable multiple selection.
```
